docs/en/stack/security/troubleshooting.asciidoc

```diff
@@ -392,6 +392,123 @@ Kerberos/SPNEGO debug logging on JVM, add following JVM system properties:
 
 For more information about JVM system properties, see {ref}/jvm-options.html[configuring JVM options].
 
+[[trb-security-saml]]
+=== Common SAML issues
+
+*Symptoms:*
+
+. Authentication in Kibana fails and the following error is printed in the
+  Elasticsearch logs:
++
+--
+[source, shell]
+-----------------------------------------------
+Cannot find any matching realm for [SamlPrepareAuthenticationRequest{realmName=null,
+assertionConsumerServiceURL=https://my.kibana.url/api/security/v1/saml}]
+-----------------------------------------------
+--
```

Comment:
I would like the **Symptoms** and **Resolutions** to be paired in the document rather than being 2 lists.